src/test/scala/cognite/spark/v1/FunctionsUDFTest.scala

```diff
@@ -1,52 +1,52 @@
-package cognite.spark.v1
-
-import cognite.spark.v1.udf.CogniteUdfs
-import cats.effect.IO
-import cognite.spark.v1.Constants.{DefaultBaseUrl, DefaultMaxRetries, DefaultMaxRetryDelaySeconds}
-import com.cognite.sdk.scala.v1.GenericClient
-import org.apache.spark.sql.Row
-import org.scalatest.{FlatSpec, Inspectors, Matchers, ParallelTestExecution}
-import sttp.client3.SttpBackend
-
-class FunctionsUDFTest
-    extends FlatSpec
-    with Matchers
-    with ParallelTestExecution
-    with SparkTest
-    with Inspectors {
-  import CdpConnector.ioRuntime
-
-  ignore should "read assets with functionUDF" taggedAs ReadTest in {
-    implicit val backend: SttpBackend[IO, Any] =
-      CdpConnector.retryingSttpBackend(DefaultMaxRetries, DefaultMaxRetryDelaySeconds)
-    val df = dataFrameReaderUsingOidc
-      .option("type", "assets")
-      .option("limitPerPartition", "1")
-      .option("partitions", "1")
-      .option("limit", "10")
-      .load()
-
-    new CogniteUdfs(spark).initializeUdfs(writeApiKeyAuth)
-
-    val func = for {
-      client <- GenericClient.forAuth[IO](
-        Constants.SparkDatasourceVersion,
-        writeApiKeyAuth,
-        DefaultBaseUrl,
-        apiVersion = Some("playground"))
-      functions <- client.functions.read()
-    } yield functions.items.head
-    val functionId = func.unsafeRunSync().id.get
-
-    df.createOrReplaceTempView("assetsRead")
-    val res: Array[Row] = spark
-      .sql(s"""select cdf_function($functionId, \"{}\") as functionResult from assetsRead""")
-      .collect()
-
-    forAll(res) { row: Row =>
-      row.getString(0) shouldEqual "42"
-    }
-
-    assert(res.length == 1)
-  }
-}
+//package cognite.spark.v1
```

Comment:
commented since the test is already ignore, we can make it work again in the future, but need to switch CogniteUdfs to use OIDC instead of API Key